layouts/shortcodes/featured/donate-prefill-lg.html

```diff
@@ -0,0 +1,20 @@
+{{ $wrapperClass := .Get "wrapperClass" | default "w-[calc(100vw - (100vw - 100%))] my-10 -mx-[50%]"}}
```

Comment:
Does this wrapper class work? ISTM, that Tailwind need the spaces to be underscores.